packages/regexp-split/index.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+const { Observable } = require("rxjs");
+
+module.exports = ({ value, params }) => {
+  return new Observable.create(subscriber => {
+    try {
+      const values = value.split(params.pattern);
+
+      values.forEach(v => {
+        subscriber.next(v);
+      });
```

Comment:
Will an error thrown by a subscriber break out of the forEach?

Reply:
No. The observer will no longer yield values to subscribers but the forEach continues to run. Replaced this with a while that breaks when the observer completes or errors.